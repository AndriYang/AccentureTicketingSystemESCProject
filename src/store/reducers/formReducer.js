const initState = {

}

const formReducer = (state= initState, action) => {
  switch(action.type){

    case 'CREATE_FORM':
      console.log('created form', action.form);
      return state;

    case 'CREATE_FORM_ERROR':
      console.log('create project error', action.err);
      return state;

    default:
      return state;
  }
};

export default formReducer;
