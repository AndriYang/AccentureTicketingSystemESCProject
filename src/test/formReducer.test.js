import formReducer from '../store/reducers/formReducer'

describe('With snapshots', () => {
  it('should return initial state', () =>{
    expect(formReducer(undefined, {})).toMatchSnapshot();
  })

  it('should handle CREATE_FORM', ()=>{
    const action = {
      type: 'CREATE_FORM',
    }
    expect(formReducer(undefined, {})).toMatchSnapshot();
  })

  it('should handle CREATE_FORM_ERROR', ()=>{
    const action = {
      type: 'CREATE_FORM_ERROR',
    }
    expect(formReducer(undefined, {})).toMatchSnapshot();
  })
})
