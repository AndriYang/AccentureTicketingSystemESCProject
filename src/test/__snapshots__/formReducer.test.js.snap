// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With snapshots should handle CREATE_FORM 1`] = `Object {}`;

exports[`With snapshots should handle CREATE_FORM_ERROR 1`] = `Object {}`;

exports[`With snapshots should return initial state 1`] = `Object {}`;
